src/index.html

```diff
@@ -5,8 +5,355 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container container--align_center">
+      <div class="product-card container__card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac"
+          class="product-card__image"
+        >
+  
+        <div class="product-card__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+  
+        <div class="product-card__id">Код товара: 195434</div>
+  
+        <div class="product-card__rate">
+          <div class="product-card__stars rate">
+            <div class="rate__star rate__star--filled"></div>
+            <div class="rate__star rate__star--filled"></div>
+            <div class="rate__star rate__star--filled"></div>
+            <div class="rate__star rate__star--filled"></div>
+            <div class="rate__star rate__star--unfilled"></div>
+          </div>
```

Comment:
```suggestion
          <div class="product-card__stars rate rate--4">
            <div class="rate__star"></div>
            <div class="rate__star"></div>
            <div class="rate__star"></div>
            <div class="rate__star"></div>
            <div class="rate__star"></div>
          </div>
```
Переиспользуй код из layout_stars